lib/settings.rb

```diff
@@ -1,4 +1,13 @@
 class Settings
+  
+  def self.get_es_index_name
+    'claim_reviews'+(self.in_integration_test_mode? ? "_test" : "")
```

Comment:
We should not be needing this, since `es_index_name` is already a config entry. In the test config, we can simply set `es_index_name` to whatever we choose. This way we can remove `RUN_INTEGRATION_TESTS` and `in_integration_test_mode` and give freedome to Ci/CD to choose appropriate names. Also notice that this function does not read the config entry.